amida-tech/indaba-client
# Styling/inba 286 wizard subjects

#### What's this PR do?
Styles the Indaba Wizard-subjects page 
#### Related JIRA tickets:
https://jira.amida-tech.com/browse/INBA-286?filter=-1
#### How should this be manually tested?
-Go to http://localhost:3000/create-new-project
-Navigate to the subjects tab
-Click inside the table.
-Enter one or multiple subjects seperated by a comma
-Click icon or Press Enter(Test Both Please)
-The entries should appear in a zebra stripe table blow the search bar
Compare to the comps here: https://projects.invisionapp.com/d/main#/console/10818602/235698122/preview
#### Any background context you want to provide? 
The wizard footer styling will be on a separate ticket.
#### Screenshots (if appropriate):
